lib/bolt/inventory/inventory2.rb

```diff
@@ -78,41 +87,6 @@ def add_to_group(targets, desired_group)
         end
       end
 
-      def set_var(target, key, value)
-        data = { key => value }
-        set_vars_from_hash(target.name, data)
-      end
-
-      def vars(target)
-        @target_vars[target.name] || {}
-      end
-
-      def add_facts(target, new_facts = {})
-        @logger.warn("No facts to add") if new_facts.empty?
-        set_facts(target.name, new_facts)
-      end
-
-      def facts(target)
-        @target_facts[target.name] || {}
-      end
-
-      def set_feature(target, feature, value = true)
-        @target_features[target.name] ||= Set.new
-        if value
-          @target_features[target.name] << feature
-        else
-          @target_features[target.name].delete(feature)
-        end
-      end
-
-      def features(target)
-        @target_features[target.name] || Set.new
-      end
-
-      def plugin_hooks(target)
-        @target_plugin_hooks[target.name] || {}
-      end
-
       def data_hash
         {
           data: {},
```

Comment:
Are we leaving this be for now to be worked on later? Should it return the `@target` hash?

Reply:
So this is used to reconstruct inventory to support target functions in apply blocks. We intentionally set this to empty data for now (the config stuff is required or get_targets will blow up). But basically the idea is that if you use get_targets in an apply block you wont get anything from inventory v2, but you can still get new v1 targets. I think we are going to drop support for get_targets etc in apply blocks in the future.